deploy/cluster_role_binding.yaml

```diff
@@ -0,0 +1,13 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: benchmark-operator
+subjects:
+- kind: ServiceAccount
+  name: benchmark-operator
+  #FIXME: This shouldn't be a hard-coded reference to a namespace
+  namespace: benchmark
```

Comment:
Is this necessary? Looking at the default role_binding, I don't see a reference to the namespace?

https://github.com/jtaleric/benchmark-operator/blob/master/deploy/role_binding.yaml#L5-L7

Reply:
The ServiceAccount is in the context of a namespace. The role, service_account, and role_binding yaml files currently in deploy/ are all run in the context of the current namespace. If you create a 'benchmark' namespace, per the current README, then they will be created in that context.

This cluster_role_binding needs to attach to a ServiceAccount in that account's correct namespace, so the namespace key is required here. But these are run _before_ the operator is deployed, so there is no {{ meta.namespace }} var available to use here. I'm currently not sure of the best way to fix this if the cluster_role_binding does need to stay here and not in the code (to be reviewed further below in this PR).

Reply:
OK, I was right in the first place -- There is an order of operations problem. In order for the benchmark operator to orchestrate another operator that itself needs cluster roles and bindings, the benchmark-operator service account needs some permissions at the cluster level to bootstrap things. Because of this, we need to fist grant those permissions as the system:admin user.

Reply:
@dustinblack maybe namespace: default would be able to set it to use default namespace?